### PR TITLE
Update tidehunter to 1.4.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`tidehunter`](https://bioconda.github.io/recipes/tidehunter/README.html): **1.3.0** &rarr; **1.4.0**

[install with bioconda](http://bioconda.github.io/recipes/tidehunter/README.html) [Conda](https://anaconda.org/bioconda/tidehunter/files)

Info | Link
-----|-----
Recipe | [`recipes/tidehunter`](https://github.com//bioconda/bioconda-recipes/tree/bump/tidehunter/recipes/tidehunter) (click to view/edit other files)
Releases |[https://github.com/yangao07/TideHunter/releases](https://github.com/yangao07/TideHunter/releases)
Author | `@yangao07`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by bgruening** [comment]
Please do not merge, this looks not correct.

File: recipes/tidehunter/meta.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ requirements:
     - {{ compiler('c') }}
   host:
     - zlib
+  run:
+    - zlib
```
Comment (dpryan79) [inline recipes/tidehunter/meta.yaml:21]: This makes no sense

**Comment by yangao07** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/tidehunter/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/tidehunter/meta.yaml:21]: zlib can be deleted here ... zlib has a run-dependency defined and it is not be needed in the run-section.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**